app/master/build.py

```diff
@@ -40,21 +39,15 @@ def __init__(self, build_request):
 
         self._error_message = None
         self.is_prepared = False
+        self._setup_is_started = False
```

Comment:
This is a temporary flag -- I'll get rid of it in another change in favor of using the state machine.